core/job/dto/validation.go

```diff
@@ -0,0 +1,19 @@
+package dto
+
+import (
+	"github.com/goto/optimus/core/job"
+	"github.com/goto/optimus/core/tenant"
+)
+
+type ValidateRequest struct {
+	Tenant       tenant.Tenant
+	JobSpecs     []*job.Spec
+	JobNames     []string
+	DeletionMode bool
+}
+
+type ValidateResult struct {
+	Name     string
```

Comment:
Calling it `stage` makes it as if there is a sequence of validations, personally. While the sequence can either be in orderly manner or not. We can change it to that instead of `name`.